functionnal_tests/cypress/integration/generic_frontend/dropdown_menu__in_content_list_spec.js

```diff
@@ -0,0 +1,110 @@
+const filename = 'Linux-Free-PNG'
+const fullFilename = 'Linux-Free-PNG.png'
+const contentType = 'image/png'
+const TITLE = 'Title'
+
+const EDIT_LABEL = 'Edit'
+const ARCHIVE_LABEL = 'Archive'
+const DELETE_LABEL = 'Delete'
+
+context('Content list -', function () {
+  beforeEach(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    
+  })
+  describe('dropdown menu -', function() {
```

Comment:
I don't think mixing context and describe is a good idea

most of other tests uses describe, I'd advise to stick to this